internal/wasm/sdk/opa/pool.go

```diff
@@ -129,14 +129,22 @@ func (p *pool) SetPolicyData(policy []byte, data []byte) error {
 	p.mutex.Lock()
 
 	if !p.initialized {
-		vm, err := newVM(policy, data, p.memoryMinPages, p.memoryMaxPages)
+		var err error
+		for i := 0; i < len(p.available); i++ {
+			var vm *vm
+			vm, err = newVM(policy, data, p.memoryMinPages, p.memoryMaxPages)
+			if err == nil {
+				p.vms = append(p.vms, vm)
+				p.acquired = append(p.acquired, false)
+			} else {
+				err = fmt.Errorf("%v: %w", err, ErrInvalidPolicyOrData)
+				break
+			}
+		}
```

Comment:
On my desktop, startup for a trivial bundle is several seconds (because this has to initialize one VM per CPU thread...) I'm tempted to just throw goroutines at this to make startup more response. WDYT?

Reply:
Done